lib/compiler/doc/src/compile.xml

```diff
@@ -774,11 +778,15 @@ module.beam: module.erl \
       <name since="">forms(Forms, Options) -> CompRet</name>
       <fsummary>Compiles a list of forms.</fsummary>
       <type>
-        <v>Forms = [Form]</v>
+        <v>Forms = <seetype marker="#forms">forms()</seetype></v>
+        <v>forms() = [<seetype marker="erl_parse#abstract_form">erl_parse:abstract_form</seetype>] | <seetype marker="cerl#type-c_module">cerl:c_module()</seetype></v>
```

Comment:
```suggestion
        <v>forms() = [<seetype marker="erl_parse#abstract_form">erl_parse:abstract_form</seetype>] | <seeerl marker="cerl#type-c_module">cerl:c_module()</seeerl></v>
```

Since cerl docs has been generated using edoc the types in it do not use types from specs, so you need to use the `<seeerl>` tag for links.

Reply:
😄 Good to know, thanks.

Reply:
I've force-pushed my last commit. Will check back again to see if tests are passing.

Reply:
Everything's green now. Thanks much.